contracts/vaults/TreasuryVault/RibbonTreasuryVaultLite.sol

```diff
@@ -0,0 +1,919 @@
+     */
+    function withdrawInstantly(uint256 amount) external nonReentrant {
+        Vault.DepositReceipt storage depositReceipt =
+            depositReceipts[msg.sender];
+
+        uint256 currentRound = vaultState.round;
+        require(amount > 0, "!amount");
+        require(depositReceipt.round == currentRound, "Invalid round");
+
+        uint256 receiptAmount = depositReceipt.amount;
+        require(receiptAmount >= amount, "Exceed amount");
+
+        // Subtraction underflow checks already ensure it is smaller than uint104
+        depositReceipt.amount = uint104(receiptAmount.sub(amount));
+        vaultState.totalPending = uint128(
+            uint256(vaultState.totalPending).sub(amount)
+        );
+
+        emit InstantWithdraw(msg.sender, amount, currentRound);
+
+        transferAsset(msg.sender, amount);
+    }
+
+    /**
+     * @notice Completes a scheduled withdrawal from a past round. Uses finalized pps for the round
+     */
+    function completeWithdraw() external nonReentrant {
+        uint256 withdrawAmount = _completeWithdraw();
+        lastQueuedWithdrawAmount = uint128(
+            uint256(lastQueuedWithdrawAmount).sub(withdrawAmount)
+        );
+    }
+
+    /************************************************
+     *  VAULT OPERATIONS
+     ***********************************************/
+
+    /*
+     * @notice Helper function that performs most administrative tasks
+     * such as setting next option, minting new shares, getting vault fees, etc.
+     * @param lastQueuedWithdrawAmount is old queued withdraw amount
+     * @return newOption is the new option address
+     * @return lockedBalance is the new balance used to calculate next option purchase size or collateral size
+     * @return queuedWithdrawAmount is the new queued withdraw amount for this round
+     */
+    function _rollToNextOption(uint256 lastQueuedWithdrawAmount)
+        internal
+        returns (
+            address newOption,
+            uint256 lockedBalance,
+            uint256 queuedWithdrawAmount
+        )
+    {
+        require(block.timestamp >= optionState.nextOptionReadyAt, "!ready");
+
+        newOption = optionState.nextOption;
+        require(newOption != address(0), "!nextOption");
+
+        uint256 currentRound = vaultState.round;
+        address recipient = feeRecipient;
+        uint256 mintShares;
+        uint256 managementFeeInAsset;
+        {
+            uint256 newPricePerShare;
+            (
+                lockedBalance,
+                queuedWithdrawAmount,
+                newPricePerShare,
+                mintShares,
+                managementFeeInAsset
+            ) = VaultLifecycleTreasury.rollover(
+                vaultState,
+                VaultLifecycleTreasury.RolloverParams(
+                    vaultParams.decimals,
+                    IERC20(vaultParams.asset).balanceOf(address(this)),
+                    totalSupply(),
+                    lastQueuedWithdrawAmount,
```

Comment:
`lastQueuedWithdrawAmount` has the same name both in `RibbonTreasuryVaultLiteStorage` and as `_rollToNextOption` function input, which is creating a conflict/shadowing.

Perhaps we could use `_lastQueuedWithdrawAmount` instead?